src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -65,13 +73,48 @@ private static object HandleClass(Type memberType, int levelsLeft)
     private static Array HandleSingleDimensionalArray(Type arrayType) =>
         Array.CreateInstance(arrayType.GetElementType(), 0);
 
+    private static object HandleString(string memberName)
+    {
+        try
+        {
+            var caseInsensitiveMember = memberName.ToLower();
+            foreach (var jObject in s_jsonData)
+            {
+                if (jObject.First is JProperty jProperty &&
+                    caseInsensitiveMember.Contains(jProperty.Name.ToLower()))
```

Comment:
Use `StringComparison.InvariantCultureIgnoreCase` instead of `ToLower()`
And consider pre-processing the `List<JObject>` object into more convenient data structure.

Reply:
Done